src/aggregate/reducers/to_list.c

```diff
@@ -53,16 +53,16 @@ static RSValue *tolistFinalize(Reducer *rbase, void *ctx) {
   TrieMapIterator *it = TrieMap_Iterate(tlc->values, "", 0);
   char *c;
   tm_len_t l;
-  void *ptr;
-  RSValue **arr = rm_calloc(tlc->values->cardinality, sizeof(RSValue));
+  RSValue *val;
+  RSValue **arr = rm_malloc(tlc->values->cardinality * sizeof(RSValue*));
```

Comment:
use `rm_calloc`
also, for cleaner code I suggest we'll have a dedicated function for this instead of direct call to `rm_malloc`

Reply:
we have bad experience with un-initilized memory :)

The function should be something like
```c
RSValue **RSVvalue_NewArray(size_t count) {
    return rm_calloc(count, sizeof(RSValue);
}
```

Let's try to avoid low-level calls where we can abstract them